src/libutil/sysutil.cpp

```diff
@@ -575,6 +564,17 @@ Sysutil::put_in_background(int argc, char* argv[])
 }
 
 
+bool
+#ifdef _WIN32
+Sysutil::put_in_background(int, char*[])
+#else
+Sysutil::put_in_background(int argc, char* argv[])
+#endif
```

Comment:
I'm not sure this is necessary. Did you get warnings on non-Windows platforms if you just stuck to `(int, char*[])` across the board?

Reply:
Running CI tests, removed legacy WIN32 arguments inherited from master.